src/components/ChatGhostUI/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import _ from 'underscore';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../withWindowDimensions';
+import ContentLoadingOneLine from './ContentLoadingElements/ContentLoadingOneLine';
+import ContentLoadingTwoLines from './ContentLoadingElements/ContentLoadingTwoLines';
+import ContentLoadingThreeLines from './ContentLoadingElements/ContentLoadingThreeLines';
+import CONST from '../../CONST';
+
+const propTypes = {
+    ...windowDimensionsPropTypes,
+};
+
+const ChatList = (props) => {
+    const windowHeight = props.windowHeight;
+
+    const height = CONST.CHAT_GHOST_ROW_HEIGHT;
+    const numberOfRows = Math.floor(windowHeight / height);
+    const contentItems = new Array(numberOfRows);
```

Comment:
```suggestion
    const possibleVisibleContentItems = new Array(Math.floor(props.windowHeight / CONST.CHAT_GHOST_ROW_HEIGHT));
```

Reply:
Add a comment why are we creating a null item array

Reply:
**suggestion:** Do something obvious like use a `for` loop so the comment won't be necessary.